src/DropboxLogin.jsx

```diff
@@ -0,0 +1,111 @@
+// Login screen that allows user to enable saving notes to their dropbox.
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+    Button,
+    CssBaseline,
+    FormControl,
+    Input,
+    InputLabel,
+    Paper,
+    Typography,
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+    main: {
+        width: 'auto',
+        display: 'block', // Fix IE 11 issue.
+        marginLeft: theme.spacing.unit * 3,
+        marginRight: theme.spacing.unit * 3,
+        [theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
+            width: 400,
+            marginLeft: 'auto',
+            marginRight: 'auto',
+        },
+    },
+    paper: {
+        marginTop: theme.spacing.unit * 8,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        padding: `${theme.spacing.unit * 2}px ${theme.spacing.unit * 3}px ${theme.spacing.unit *
+            3}px`,
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing.unit,
+    },
+    submit: {
+        marginTop: theme.spacing.unit * 3,
+    },
+});
+
+const defaultAccessToken = process.env.REACT_APP_DROPBOX_ACCESS_TOKEN || '';
```

Comment:
This is being defined in `userConfig` also?

Reply:
Yeah. Forgot to remove and use the config one.